paasta_tools/kubernetes_tools.py

```diff
@@ -1349,6 +1363,48 @@ def get_node_affinity(self) -> Optional[V1NodeAffinity]:
             required_during_scheduling_ignored_during_execution=selector,
         )
 
+    def get_pod_anti_affinity(self) -> Optional[V1PodAntiAffinity]:
+        """
+        Converts the given anti-affinity on service and instance to pod
+        affinities with the "paasta.yelp.com" prefixed label selector
+        :return:
+        """
+        conditions = self.config_dict.get("anti_affinity", [])
+        if not conditions:
+            return None
+
+        if not isinstance(conditions, list):
+            conditions = [conditions]
+
+        affinity_terms = []
+        for condition in conditions:
+            label_selector = self._kube_affinity_condition_to_label_selector(condition)
+            if label_selector:
+                affinity_terms.append(
+                    V1PodAffinityTerm(
```

Comment:
I added the namespace key in the doc for the sake of completeness. I am not seting the `Namespace` explicitly here because we want to schedule it in the `pod's namespace` and as per the [kubernetes source code](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L2783) a `null or empty` means the "pod's namespace"

Reply:
nice, thx for clarifying